pom.xml

```diff
@@ -31,9 +31,9 @@
     <properties>
         <revision>1.26</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.190.1</jenkins.version>
+        <jenkins.version>2.190.3</jenkins.version>
         <java.level>8</java.level>
-        <pipeline-model-definition-plugin.version>1.5.1</pipeline-model-definition-plugin.version>
+        <pipeline-model-definition-plugin.version>1.8.1</pipeline-model-definition-plugin.version>
```

Comment:
Probably could be removed at this point, picking up `bom` version. Just have not checked recently.